rankboost.py

```diff
@@ -436,6 +436,42 @@ def prepare_dataframe(mhc, predfile, expfile, pepfile, efilt_threshold, boost):
     _log.info('Finished creating the data frame for rank boosting..')
     return all_data, stats, boost
 
+def collapse_iars (stats):
+    """
+    Collapse duplicated IARs.
+    :param stats:
```

Comment:
:param pandas.core.frame.DataFrame stats: The stats dataframe generated from parsing the input calls